src/gui.py

```diff
@@ -382,18 +394,108 @@ def mousePressEvent(self, mouse_ev):
         if mouse_ev.button() == qc.Qt.RightButton:
             self.right_click_menu.exec_(qg.QCursor.pos())
 
+    def __del__(self):
+        print('Spectrogram deleted')
```

Comment:
Would be more elegant to use
`logger.debug('Spectrogram deleted')`

Reply:
Agreed. If I change this now, will it still be included in the merge request?

Reply:
It would be included if it got pushed to the same branch of this merge request. But @sebastianrosenzweig open the MR. Nevermind. Let's ignore this here. `black` will take care of that. Let's fix this in another MR.

Reply:
Ah sorry I was confused which thread I was adding this comment to :)
The print statement needed to be fixed in the same branch. @sebastianrosenzweig if you are too busy finishing your work at the moment you can also convert each thread into an issue so that we can fix this later.

Reply:
Fixed this in recent commit